v0/destinations/posthog/data/PHPropertiesConfig.json

```diff
@@ -37,30 +37,20 @@
   },
   {
     "destKey": "$time",
-    "sourceKeys": [
-      "originalTimestamp",
-      "timestamp"
-    ]
+    "sourceKeys": ["originalTimestamp", "timestamp"]
```

Comment:
we can add it from generic field mapping. Can you check for the other implementations and see how to do it.

Reply:
i don't get this. could you please explain bit more @shrouti1507 .